testsuite/features/support/lavanda.rb

```diff
@@ -158,8 +158,8 @@ def os_version
   #   successcodes: An array with the values to be accepted as success codes from the command run.
   #   buffer_size: The maximum buffer size in bytes. Defaults to 65536.
   #   verbose: Whether to log the output of the command in case of success. Defaults to false.
-  def run(cmd, separated_results: false, check_errors: true, timeout: DEFAULT_TIMEOUT, user: 'root', successcodes: [0], buffer_size: 65_536, verbose: false)
-    cmd_prefixed = @in_has_mgrctl ? "mgrctl exec -i '#{cmd.gsub(/'/, '\'"\'"\'')}'" : cmd
+  def run(cmd, mgrctl_cmd: 'exec', separated_results: false, check_errors: true, timeout: DEFAULT_TIMEOUT, user: 'root', successcodes: [0], buffer_size: 65_536, verbose: false)
```

Comment:
With that change, you should also update the argument list in the comment above and add it there, too.
I also wonder if this makes sense at all, since in the line below

```ruby
cmd_prefixed = @in_has_mgrctl ? "mgrctl #{mgrctl_cmd} -i '#{cmd.gsub(/'/, '\'"\'"\'')}'" : cmd
```
we have a static `-i` flag after the `mgrctl_cmd` command. According to the `mgrctl` help, this flag is only available with the `exec` command. The other commands (`api` and `cp`) do not have this.

Reply:
This is a leftover, I handled differently at the end.
YAGNI, I will remove this parameter.